spec/feature/hearings/virtual_hearings/hearing_details_spec.rb

```diff
@@ -121,576 +120,550 @@ def ensure_link_present(link, disable)
     expect(page).to have_selector(:css, "a[href='#{link}']") unless disable
   end
 
-  context "initial hearing type is Video" do
-    scenario "email notification history is not displayed" do
-      visit "hearings/" + hearing.external_id.to_s + "/details"
+  shared_context "shared behaviors" do
```

Comment:
```suggestion
  shared_examples "shared behaviors" do
```
Prefer `shared_examples` when you're sharing code that has expectations. `shared_context` is for sharing setup. They're functionally identical but still useful to use consistently (like `describe` and `context`)